tests/testthat/test-hintr-plot-data.R

```diff
@@ -52,3 +52,22 @@ test_that("there is metadata for every indicator in comparison data", {
                            metadata$plot_type == "barchart", ]
   expect_setequal(indicators, comparison$indicator)
 })
+
+test_that("hintr data can be saved and read as qs or duckdb type", {
+  t_qs <- tempfile(fileext = ".qs")
+  t_db <- tempfile(fileext = ".duckdb")
+  d <- data.frame(x = c(1, 2, 3), y = c(4, 5, 6))
+
+  hintr_save(d, t_qs)
+  hintr_save(d, t_db)
+  expect_equal(read_hintr_output(t_qs), read_hintr_output(t_db))
```

Comment:
should we not test each case against the concrete dataframe because if there is a similar bug in both in the future then this test could still pass